board/model_manager.py

```diff
@@ -118,14 +131,63 @@ def sample_data_and_train(self):
+        if self.recommender_name=="bpr":
+            print("recommender chosen is:", self.recommender_name)
+            recommender_model = BPR(batch_size=1000, 
+                            total_users=train_dataset.total_users(), 
+                            total_items=train_dataset.total_items(), 
+                            dim_user_embed=50, 
+                            dim_item_embed=50, 
+                            save_model_dir=model_save_dir, 
+                            train=True, serve=True)
+        elif self.recommender_name=="drr":
+            print("recommender chosen is:", self.recommender_name)
+            recommender_model = DRR(batch_size=1000, 
+                            total_users=train_dataset.total_users(), 
+                            total_items=train_dataset.total_items(), 
+                            dim_user_embed=50, 
+                            dim_item_embed=50, 
+                            save_model_dir=model_save_dir, 
+                            train=True, serve=True)
+        elif self.recommender_name=="pmf":
+            print("recommender chosen is:", self.recommender_name)
+            recommender_model = PMF(batch_size=1000, 
+                            total_users=train_dataset.total_users(), 
+                            total_items=train_dataset.total_items(), 
+                            dim_user_embed=50, 
+                            dim_item_embed=50, 
+                            save_model_dir=model_save_dir, 
+                            train=True, serve=True)
+        elif self.recommender_name=="rnnrec":
+            print("recommender chosen is:", self.recommender_name,"BUT STILL USING BPR")
+            print("So sorry! we need you to give us the max_seq_len, num_units as well, so switching to BPR")
+            recommender_model = BPR(batch_size=1000, 
+                            total_users=train_dataset.total_users(), 
+                            total_items=train_dataset.total_items(), 
+                            dim_user_embed=50, 
+                            dim_item_embed=50, 
+                            save_model_dir=model_save_dir, 
+                            train=True, serve=True)
+        elif self.recommender_name=="ucml":
+            print("recommender chosen is:", self.recommender_name)
+            recommender_model = UCML(batch_size=1000, 
+                            total_users=train_dataset.total_users(), 
+                            total_items=train_dataset.total_items(), 
+                            dim_user_embed=50, 
+                            dim_item_embed=50, 
+                            save_model_dir=model_save_dir, 
+                            train=True, serve=True)
+        elif self.recommender_name=="vbpr":
+            print("recommender chosen is:", self.recommender_name,"BUT STILL USING BPR")
+            print("So sorry! we need you to give us the dim_v as well, so switching to BPR")
```

Comment:
These comments will be removed post completion for model_algorithm-evaluator-sampler validation in next PR